apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -272,6 +329,27 @@ type ResourceIdentifier struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
+// ResourcePlacementStatus represents the status of a group of resource placed on selected cluster
+// according to the PlacementPolicy field in the ClusterResourcePlacement object.
+type ResourcePlacementStatus struct {
+	// Conditions is an array of current observed conditions for ResourcePlacementStatus.
+	// +optional
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// ClusterName is the name of the ManagedCluster. If it is not empty, its value should be unique cross all
+	// placement decisions for the Placement.
+	// +kubebuilder:validation:Required
+	// +required
+	ClusterName string `json:"clusterName"`
```

Comment:
Move this to be before conditions. The subject should appear first instead of the attributes of the subject.